crates/bitwarden-vault/src/sync.rs

```diff
@@ -54,6 +54,7 @@ pub(crate) async fn sync(client: &Client, input: &SyncRequest) -> Result<SyncRes
 
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
+/// Data returned from `/accounts/profile`.
```

Comment:
This is false. `ProfileResponseModel` from `bitwarden-api-api` is the response from the endpoint. This is a part of the `SyncResponse` which is the internal SDK response from performing a sync.